.github/workflows/publish.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout Package
-        uses: actions/checkout@master
+        uses: actions/checkout@v2
```

Comment:
Just want to confirm, why did we change this from `master` branch to `default` branch